### ISSUE TITLE
Set accessible default Font

### ISSUE THREAD

**Issue by ElectronicBlueberry** [galaxyproject/galaxy#15061]
This issue is intended as a discussion issue on the topic of changing Galaxys default font.
See this PR for a preview of the suggested changes: #15062

---

Currently galaxy uses a font stack which selects a suitable default font per system, and falls back to the systems sans-serif font if none is found. This approach is reasonable, as we do not have to ship our own font. However, many system sans-serif fonts can be hard to read for people with vision impairment. It also leads to inconsistencies between systems regarding galaxies default appearance.

For these reasons, I propose shipping an accessible modern well-legible default font with galaxy instead of relying on system fonts.

## Legibility

Example using the following characters: Ili!O0o
Here is what these look like in several variants of Galaxys fonts:

SF Pro (Mac) 
![image](https://user-images.githubusercontent.com/44241786/203994691-6c081721-e016-4849-a7de-35847da00fd4.png)

Seoge UI (Windows)
![image](https://user-images.githubusercontent.com/44241786/203994587-0987a5b7-e9c1-4d0c-9822-8347de14fd6b.png)

Liberation Sans (Ubuntu)
![image](https://user-images.githubusercontent.com/44241786/203995005-8b53b52c-6fa7-4737-b2f7-acab0da7b89d.png)

Some of these symbols are hard to tell apart, even with clear vision. With vision impairment, text can become a lot more challenging to read. With dyslexia, similar looking characters also increase the reading difficulty, and can make reading slower.

## Atkinson Hyperlegible

Often serif, or comic fonts are used as more legible alternatives to sans-serif fonts. These however can look dated or tacky. Luckily the Braille Institute has developed a very legible sans-serif font, [Atkinson Hyperlegible](https://brailleinstitute.org/freefont).

The font is free to use with a [permissive license](http://brailleinstitute.org/wp-content/uploads/2020/11/Atkinson-Hyperlegible-Font-License-2020-1104.pdf).

Here is the example from before in this font:
![image](https://user-images.githubusercontent.com/44241786/203996205-05dd72c8-7ed9-4b86-a29c-44b45365b373.png)

See the [Braille Institutes Page](https://brailleinstitute.org/freefont) on the font for more examples.

Every character is easy to tell apart at a glance, making texts more readable for people with reading and vision impairments.

## Why default

Setting this font as a site wide default ensures people can make use of it on machines on which they may not be able to override the browsers fonts, or lack the technical knowledge to do so. Changing the browsers fonts can also be undesirable, as some pages may not work well with different fonts.

It also leads to future training material including this font in screenshots and videos, increasing their accessibility as a side-effect.

## Consistency

Having the fonts consistent across systems can allow us to tweak font sizes and weights to a finer degree, to ensure clear and properly sized texts on all systems.

**Comment by bgruening** [comment]
Thanks Laila for writing this up and working on it!
